src/web/TextInput.tsx

```diff
@@ -122,12 +122,29 @@ export class TextInput extends React.Component<Types.TextInputProps, TextInputSt
                     onMouseUp={ this._checkSelectionChanged }
                     onPaste={ this._onPaste }
                     aria-label={ this.props.accessibilityLabel }
-                    type={ this.props.secureTextEntry ? 'password' : 'text' }
+                    type={ this.props.secureTextEntry ? 'password' : this._keyboardType() }
                 />
             );
         }
     }
 
+    private _keyboardType = () => {
```

Comment:
Should be called _getKeyboardType. (All methods should generally begin with verb.)